BST/BinaryTreeFromLowToHigh.py

```
@@ -0,0 +1,44 @@
+class TreeNode:
+      def __init__(self,val:int=0):
+          self.val=val
+          self.left=None 
+          self.right=None 
+class Solution:
+        def __init__(self):
+            self.array=[]
+            self.root=None
+        def preOrder(self,root:TreeNode):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 8 spaces, expected 4

(W0311)